content/deploy/backup/graphdb.md

```diff
@@ -11,50 +10,75 @@ menu:
     identifier:
 ---
 
-This guide shows you the procedure to back up the Rhize Graph database.
+This guide shows you how to back up the Rhize Graph database.
+You can also use it to model an automation workflow.
 
 ## Prerequisites
 
 Before you start, ensure you have the following:
 
-- Access to the [Rhize Kubernetes Environment]({{< relref "../install/configure-kubernetes" >}})
-- [kubectl](https://kubernetes.io/docs/tasks/tools/) and [kubectx](https://github.com/ahmetb/kubectx)
-- Optional: the [k8 Lens IDE](https://k8lens.dev), if you prefer to manage Kubernetes graphically
+- A designated backup location, for example `~/rhize-backups/database`.
+- Access to your [Rhize Kubernetes Environment]({{< relref "install" >}})
+{{% param pre_reqs %}}.
 
-## Procedure
+Also, before you start, confirm you are in the right context and namespace.
 
-First, confirm you are in the right context and namespace.
+```bash
+## context
+kubectl config current-context
+## namespace
+kubectl get namespace
+```
+## Steps
 
-    ```bash
-    ## context
-    kubectl config current-context
-    ## namespace
-    kubectl get namespace
-    ```
-Record these outputs for record keeping.
+To back up the database, follow these steps:
 
-Then, to back up the database, follow these steps:
-    
 1. Check the logs for the alpha and zero pods, either in Lens or with [`kubectl logs`](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#logs).
     Ensure there are no errors.
 
 1. Open a pod shell for one of the alpha pods.
 
-1. Use curl to create a backup of the node.
+    ```bash
+    kubectl exec {{< param application_name >}}-baas-baas-alpha-0 -- /bin/bash
+    ```
+
+    For details, read the Kubernetes topic [Get Shell to a Running Container](https://kubernetes.io/docs/tasks/debug/debug-application/get-shell-running-container/).
+
+1. Use `curl` or something similar to create a backup of the node:
 
     ```bash
     curl --location --request POST 'http://localhost:8080/admin' \
     --header 'Content-Type: application/json' \
-    --data-raw '{"query":"mutation {\r\n export(input: {format: \"json\", destination: \"/dgraph/backups/'"$(date +"%Y-%m-%dT%H.%M.%SZ")"'\"}) {\r\n response {\r\n message\r\n code\r\n }\r\n}\r\n}","variables":{}}'
+    --data-raw '{"query":"mutation {\r\n export(input: {format: \"json\", destination: \"/dgraph/backups/'" $(date +"%Y-%m-%dT%H.%M.%SZ")"'\"}) {\r\n response {\r\n message\r\n code\r\n }\r\n}\r\n}","variables":{}}'
     ```
 
```

Comment:
I like this better but it requires jq as a dependency :thinking:

Reply:
I decided to keep it two steps, just because it might be a little easier to understand. Once I add a collapsible (#19) feature to, I'll include this single option . https://github.com/libremfg/libremfg.github.io/pull/23/commits/850a88b53b63f56fedd5ff5bff2d497227b15875